src/ontology/mondo-edit.obo

```diff
@@ -497558,7 +497560,7 @@ xref: OMIM:619637 {source="MONDO:equivalentTo"}
 xref: UMLS:C5562029 {source="MEDGEN:1794239", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0044807 {source="OMIM:619637"} ! inherited dystonia
 is_a: MONDO:0100617 {source="https://orcid.org/0000-0001-5208-3432"} ! VPS11-related neurological disorder
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/14583 {source="OMIM:619637"} ! VPS11
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/14583 ! VPS11
```

Comment:
the removed relationship between the phenotype and gene is provisional https://omim.org/entry/619637